homeassistant/components/xiaomi_miio/air_quality.py

```diff
@@ -0,0 +1,192 @@
+"""Support for Xiaomi Mi Air Quality Monitor (PM2.5)."""
+from miio import AirQualityMonitor, DeviceException
+import voluptuous as vol
+
+from homeassistant.components.air_quality import (
+    AirQualityEntity,
+    PLATFORM_SCHEMA,
+    _LOGGER,
+    ATTR_PM_2_5,
+)
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TOKEN, ATTR_TEMPERATURE
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+
+DEFAULT_NAME = "Xiaomi Miio Sensor"
+DATA_KEY = "air_quality.xiaomi_miio"
+
+ATTR_CO2E = "carbon_dioxide_equivalent"
+ATTR_HUMIDITY = "relative_humidity"
+ATTR_TVOC = "total_volatile_organic_compounds"
+ATTR_MANUFACTURER = "manufacturer"
+ATTR_MODEL = "model"
+ATTR_SW_VERSION = "sw_version"
```

Comment:
I think some of these constants are available in common consts file?

Reply:
I couldn't find any!